data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/schema/getPoolAttestationsV2Response.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title" : "getPoolAttestationsV2Response",
+  "type" : "object",
+  "required" : [ "version", "data" ],
+  "properties" : {
+    "version" : {
+      "type" : "string",
+      "enum" : [ "phase0", "altair", "bellatrix", "capella", "deneb", "electra" ]
+    },
+    "data" : {
+      "type" : "array",
+      "items" : {
+        "title" : "Attestation",
+        "type" : "object",
+        "oneOf" : [ {
+          "$ref" : "#/components/schemas/AttestationPhase0"
+        }, {
+          "$ref" : "#/components/schemas/AttestationPhase0"
+        }, {
+          "$ref" : "#/components/schemas/AttestationPhase0"
+        }, {
+          "$ref" : "#/components/schemas/AttestationPhase0"
+        }, {
+          "$ref" : "#/components/schemas/AttestationPhase0"
+        }, {
+          "$ref" : "#/components/schemas/AttestationElectra"
+        } ]
```

Comment:
I am currently working on a way to fix this issue where we get the duplicate schemas in our openapi def.

Reply:
Actually @rolfyone fixed it here https://github.com/Consensys/teku/pull/8440

Reply:
Fixed thanks